macros/ano_macros.cfg

```diff
@@ -1008,35 +1008,18 @@
 #enddef
 
 #define MAKE_LOYAL OP WHAT
-    {CLEAR_VARIABLE ano_loyal}
-    [store_unit]
-        variable=ano_loyal
+    [modify_unit]
         [filter]
             {OP}={WHAT}
             side=1
         [/filter]
-    [/store_unit]
+		{TRAIT_LOYAL}
+    [/modify_unit]
     {MSG_narrator _"The unit $ano_loyal[0].name became LOYAL."}
-    {VARIABLE (ano_loyal[0].traits_description) ("$ano_loyal[0].traits_description" + _"loyal")}
-    {VARIABLE (ano_loyal[0].upkeep) ("loyal")}
-    {VARIABLE (ano_loyal[0].modifications.trait[2].id) ("loyal")}
-    {VARIABLE (ano_loyal[0].modifications.trait[2].name) (_"loyal")}
-    {VARIABLE (ano_loyal[0].modifications.trait[2].effect.apply_to) ("loyal")}
-    # wmllint: validate-off
-    {VARIABLE (ano_loyal[0].modifications.trait[2].description) (_"Zero upkeep")} # wmllint: ignore
-    # wmllint: validate-on
-    {VARIABLE (ano_loyal[0].overlays) ("misc/loyal-icon.png")}
-    [unstore_unit]
-        variable=ano_loyal[0]
-    [/unstore_unit]
-    {CLEAR_VARIABLE ano_loyal}
 #enddef
 
 # FIXME: how about refactoring this just to use the normal "notes" tag?
 #define OBJECTIVE_NOTES _NOTES_TEXT
     note="*"+_"Scenario notes:"+"
 "+{_NOTES_TEXT}
 #enddef
-
-# For the AI controller:
```

Comment:
uh why did this part get removed?

Reply:
Oh, sorry. I hadn't realized that subfolders have to be included (just tested, and I got a MACRO error)

Reply:
ok well it looks like [9aa124d](https://github.com/nemaara/A_New_Order/pull/108/commits/9aa124d76232774fd25d7b1ec9473b5bcfcd57fd) fixes this, so I'll resolve this conversation